lib/takeaway.rb

```diff
@@ -0,0 +1,47 @@
+require_relative ('text')
+
+
+class Takeaway
+
+  attr_accessor :order
+
+  def initialize
+    @menu = ['chips' => 2.00, 'snackbox' => 3.50, 'battered sausage' => 1.50,
+      'curry sauce' => 1.00, 'battered cod' => 3.00]
+      @ordered_items = []
+    end
+
+    def menu
+      @menu
+    end
+
+    def order(item, quantity)
+      x = 0
```

Comment:
Variable name readibility
- when in doubt, name the variable what the value represents.

Reply:
2. It could be cleaner to store this information ```name=>quantity```, so  ```{ "pizza" => 3 }```, instead of using a while loop, incrementing a counter, and storing copies of the item